### PR TITLE
Handle one new email when receiving push notification

### PR DESCRIPTION
**Pull request by Hirobreak**
- Receiving a new email push notification retrieve that single event and parses it
- Works with background accounts

### CODE REVIEW

File: electron_app/src/ipc/manager.js [hunk 1]
```diff
@@ -119,11 +119,12 @@ ipc.answerRenderer('fs-delete-email-content', async params => {
 });
 
 ipc.answerRenderer('db-create-email', async params => {
+  const [account] = await dbManager.getAccountByParams({id: params.accountId})
   await fileUtils.saveEmailBody({
     body: params.body,
     headers: params.headers,
     metadataKey: parseInt(params.email.key),
-    username: getUsername()
+    username: account.recipientId.indexOf('@') > -1 ? account : `${account.recipientId}@${APP_DOMAIN}`
```
Comment (JulianAdams4) [inline electron_app/src/ipc/manager.js]: If you prefer, you can use `account.recipientId.includes('@')` instead `indexOf()`

Comment (erikaperugachi) [inline electron_app/src/ipc/manager.js]: check the true condition... ? account.recipientId

File: email_mailbox/src/utils/electronEventInterface.js [hunk 2]
```diff
@@ -106,13 +108,13 @@ const stopGettingEvents = () => {
...
 
-const parseAndStoreEventsBatch = async ({ events, hasMoreEvents }) => {
+const parseAndStoreEventsBatch = async ({ events, hasMoreEvents, accountId, optionalToken }) => {
   labelIdsEvent = new Set();
   threadIdsEvent = new Set();
   badgeLabelIdsEvent = new Set();
   labelsEvent = {};
   avatarHasChanged = false;
-
+  console.log(accountId)
```
Comment (erikaperugachi) [inline email_mailbox/src/utils/electronEventInterface.js]: delete

File: electron_app/src/ipc/mailbox.js [hunk 3]
```diff
@@ -119,14 +119,28 @@ ipc.answerRenderer(
...
-    showWindows();
-    if (threadId)
-      mailboxWindow.send('open-thread-by-notification', { threadId });
-  };
-  showNotification({ title, message, clickHandler: onClickNotification });
-});
+ipc.answerRenderer(
+  'show-notification',
+  ({ switchToAccount, title, message, threadId }) => {
+    console.log(`
```
Comment (JulianAdams4) [inline electron_app/src/ipc/mailbox.js]: Delete logs

**Pull request merged**